virocon/plotting.py

```diff
@@ -154,7 +254,29 @@ def plot_2D_isodensity(model, sample, model_desc=None, swap_axis=False, ax=None)
     return ax
 
 
+
 def plot_2D_contour(contour, sample=None, design_conditions=None, model_desc=None, swap_axis=False, ax=None):
+    """
+    Beschreibung 
+       
+    Parameters
+    ----------
+    contour: Contour
+        The environmental contour that should be plotted.
+    sample : ndarray of floats, optional
+        The environmental data sample that should be plotted against the fit.
+    design_conditions : array-like, optional
+        Specified environmental conditions under which the system must operate.
+        If None, default design conditions are computed. Defaults to None.
+    model_desc: dict, optional
+        Generated model description. Defaults to None.
```

Comment:
The description of the model. If None (the default) a generic model description will be used.